.gitchangelog.rc

```diff
@@ -0,0 +1,159 @@
+##
+## Format
+##
+##   ACTION: [AUDIENCE:] COMMIT_MSG [@TAG ...]
+##
+## Description
+##
+##   ACTION is one of 'chg', 'fix', 'new'
+##
+##       Is WHAT the change is about.
+##
+##       'chg' is for refactor, small improvement, cosmetic changes...
+##       'fix' is for bug fixes
+##       'new' is for new features, big improvement
+##
+##   SUBJECT is optional and one of 'dev', 'usr', 'pkg', 'test', 'doc'
```

Comment:
It is the name provided by default by gitconfigrc developers, it should be an error.